scripts/travis-run.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -euo pipefail
 
+if [[ $TRAVIS_BRANCH != "master" && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_REPO_SLUG == "bioconda/bioconda-recipes" ]]
+then
+    echo ""
+    echo "Tests are skipped for pushes to the main bioconda-recipes repo."
+    echo "If you have opened a pull request, please see the full tests for that PR."
+    echo "See https://bioconda.github.io/build-system.html for details"
+    echo ""
+    exit 0
+fi
+
```

Comment:
It currently seems to be impossible to hide the status of the build from github. So we either have X or ✓ and there is no way around.